test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -1517,6 +1517,13 @@ items:
     - ""
     attributeRestrictions: null
     resources:
+    - deploymentconfigs
```

Comment:
Because you're adding new roles, you need to have code in the deployment that specifically catches "forbidden" errors and treats that as "continue anyway".  Otherwise you're not backward compatible with old servers.